src/components/payment/PaymentSection.js

```diff
@@ -116,6 +117,7 @@ class PaymentSection extends React.Component {
       navigate,
       data: autoVoucherData = {},
       city,
+      financeAvailable,
```

Comment:
@ric9176 can you please update the PropTypes. Please double check that all the propTypes are in sync with the props used in the component since you are editing it.